Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml

```diff
@@ -18,6 +18,74 @@ Neos:
           host: 127.0.0.1
           port: 6379
 
+  ContentRepositoryRegistry:
+    contentRepositories:
+      test_parallel:
```

Comment:
todo we duplicate a lotta stuff here .. maybe allow multiple presets and reuse things?

Reply:
now its just 

```yaml
Neos:
  ContentRepositoryRegistry:
    contentRepositories:
      test_parallel:
        eventStore:
          factoryObjectName: Neos\ContentRepositoryRegistry\Factory\EventStore\DoctrineEventStoreFactory
        nodeTypeManager:
          factoryObjectName: Neos\ContentRepository\TestSuite\Fakes\FakeNodeTypeManagerFactory
        contentDimensionSource:
          factoryObjectName: Neos\ContentRepository\TestSuite\Fakes\FakeContentDimensionSourceFactory
        userIdProvider:
          factoryObjectName: Neos\ContentRepositoryRegistry\Factory\UserIdProvider\StaticUserIdProviderFactory
        clock:
          factoryObjectName: Neos\ContentRepositoryRegistry\Factory\Clock\SystemClockFactory
        propertyConverters: {}
        contentGraphProjection:
          factoryObjectName: Neos\ContentGraph\DoctrineDbalAdapter\DoctrineDbalContentGraphProjectionFactory
```

which is definitely ok imo!